frontend/src/metabase/ui/components/inputs/TextareaBlurChange/TextareaBlurChange.tsx

```diff
@@ -25,6 +26,7 @@ export type TextareaBlurChangeProps<
  */
 export function TextareaBlurChange<T extends TextareaProps = TextareaProps>({
   normalize = (value) => value,
+  resetOnEsc,
```

Comment:
This variable is unused, do you want to make the escape handler depend on it?

Reply:
Yeah, I forgot to use it. I updated it in a subsequent PR: https://github.com/metabase/metabase/pull/59548/files#r2152081212